selinux/README.md

```diff
@@ -1,142 +1,159 @@
-# CSI WekaFS SELinux Support
+# Add SELinux support
 
-## General Information
-When installing Weka CSI plugin on SELinux-enabled Kubernetes cluster, pods might be denied access
-to the persistent volumes provisioned on top of Weka filesystem.
+## Overview
+Security-Enhanced Linux (SELinux) is a Linux kernel security module that provides a mechanism for supporting access control security policies, including mandatory access controls (MAC).
 
-The reason behind this is a lack of permissions for containers to access objects stored on Weka cluster.
```

Comment:
have you removed this intentionally? please re-add

Reply:
Re-added this line.